sdk/include/aditof/adsd_errs.h

```diff
@@ -46,7 +46,6 @@ namespace aditof {
  * Included with aditof/camera.h
  * For example: 
  *    ADSDErrors err;
- *    LOG(INFO) << ADSDErrors::ADSD3500_STATUS_INVALID_MODE;
```

Comment:
Why the need to remove this?

Reply:
Those variables are no longer static, so we can't print them like this.